next.config.js

```diff
@@ -64,9 +65,7 @@ const nextConfig = {
                   : ""
               }`,
               "child-src 'self'",
-              `style-src 'self' ${
-                process.env.NODE_ENV === "development" ? "'unsafe-inline'" : ""
-              }`,
+              "style-src 'self' 'unsafe-inline'",
```

Comment:
question: Is this something that is for Heroku, or something we'll have to solve on prod too?

Reply:
Huh you're right, I wonder where that's being set? I don't see it in https://github.com/mozilla/blurts-server/blob/62ba006b4505d2eaf322b26bd946bc27f5d468aa/src/app.js#L109 or elsewhere (it can be overridden per-route as needed I think, I just don't see where it's happening now)

Reply:
Oh yeah, that's surprising! It looks like [helmet's setting that by default](https://www.npmjs.com/package/helmet#content-security-policy):

> `Content-Security-Policy`
> Default
>     Content-Security-Policy: default-src 'self';base-uri 'self';font-src 'self' https: data:;form-action 'self';frame-ancestors 'self';img-src 'self' data:;object-src 'none';script-src 'self';script-src-attr 'none';style-src 'self' https: 'unsafe-inline';upgrade-insecure-requests

The style-src bit: `style-src 'self' https: 'unsafe-inline'`

I do expect that it's riskier for `script-src` than for `style-src`, so hopefully Next.js [will allow a more restrictive policy](https://github.com/vercel/next.js/discussions/51039) soon.